frontend/src/Console.jsx

```
@@ -36,26 +47,70 @@ function Console(props) {
   }
 
   function toggleModal() {
-    
     setModal({ isOpen: !modalState.isOpen });
   }
 
+  function sendForm(formDetails) {
+    if (formInfo.postOrPut === "post") {
```

Since there are multiple possible methods, I would use a switch case here (even if you use the same flow for several methods, it will make it more explicit and clear)

---

Also extract this `sendForm` function to `apiUtils.js` (or `utils`/`api.js` or whatever)

---

Even though it's just two methods?

---

Yeah, I think it be more explicit, you can throw an exception on the `default` clause